main/src/main/kotlin/us/wedemy/eggeum/main/ui/MainActivity.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Designed and developed by Wedemy 2023.
+ *
+ * Licensed under the MIT.
+ * Please see full license: https://github.com/Wedemy/eggeum-android/blob/main/LICENSE
+ */
+
+/*
+* Designed and developed by Wedemy 2023.
+*
+* Licensed under the MIT.
+* Please see full license: https://github.com/Wedemy/eggeum-android/blob/main/LICENSE
+*/
+
+package us.wedemy.eggeum.main.ui
+
+import android.graphics.Color
+import android.os.Bundle
+import androidx.activity.SystemBarStyle
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.ui.setupWithNavController
+import dagger.hilt.android.AndroidEntryPoint
+import us.wedemy.eggeum.common.ui.BaseActivity
+import us.wedemy.eggeum.main.R
+import us.wedemy.eggeum.main.databinding.ActivityMainBinding
+
+// TODO 화면 위아래 하얀 공백 제거
+@AndroidEntryPoint
+class MainActivity : BaseActivity() {
+  override val binding by lazy { ActivityMainBinding.inflate(layoutInflater) }
+
+  override var statusBarStyle = SystemBarStyle.dark(Color.TRANSPARENT)
+  override var navigationBarStyle = SystemBarStyle.dark(Color.TRANSPARENT)
+
+  private val navController
+    get() = supportFragmentManager.findFragmentById(R.id.nav_host_fragment)?.findNavController()
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    initNavigation()
+  }
+
+  private fun initNavigation() {
+    binding.bnvMain.apply {
+      navController?.let { setupWithNavController(it) }
```

Comment:
```suggestion
      navController?.let(::setupWithNavController)
```

개인적으로 이런 non-null mapper 스타일은 레퍼런스 참조로 해결하는 걸 지향하는 편이에요!
tmi~~

Reply:
컴포즈에서 요런 문법을 많이 쓰던데 요기서도 적용할수있겠군요!

Reply:
> 컴포즈에서 요런 문법을 많이 쓰던데 요기서도 적용할수있겠군요!

코틀린 더블콜론 찾아보시면 도움 될 거예요!